### PR TITLE
Added some birthday splashes

### PR DESCRIPTION
**Pull request by leha-code**

### CODE REVIEW

**Comment by leha-code** [comment]
Debian works, let's see codeQL

**Comment by leha-code** [comment]
CODE QL SCANNED

File: src/mcpil.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 #  Copyright 2020 Alvarito050506 <donfrutosgomez@gmail.com>
 #  Copyright 2020 StealthHydrac/StealthHydra179/a1ma
 #  Copyright 2020 JumpeR6790
-#  Copyright 2021 Boba
```
Comment (Alvarito050506) [inline src/mcpil.py]: Uh, why? ;-;

File: src/mcpil.py [hunk 2]
```diff
@@ -332,7 +331,13 @@ def play_tab(parent):
     
     if today.month == 4 and today.day == 1:
         SPLASH = "Happy B-Day Alvarito050506"
-
+    elif today.month == 8 and today.day == 24:
+        SPLASH = "Happy Birthday LEHAtupointow"
+    elif today == 2 and today.day = 20:
+        SPLASH = "Happy Birthday Boba"
+    elif today.month == 5 and today.day == 5:
+        SPLASH = random.randint(["I shifted them a bit", "We're moving to gMCPIL or jMCPIL", "Should have come back", "YOU DON'T LIKE POTATOES?"])
```
Comment (Alvarito050506) [inline src/mcpil.py:339]: Haha

**Comment by leha-code** [comment]
sowa? mergin?

**Review by Alvarito050506 — DISMISSED** [review]

File: src/mcpil.py (hunk 1 above)
Comment (leha-code) [reply src/mcpil.py]: oh, accident i thought he changed mcpil py

File: src/mcpil.py (hunk 2 above)
Comment (leha-code) [reply src/mcpil.py:339]: yes, just your fake birthday...

File: src/mcpil.py (hunk 1 above)
Comment (leha-code) [reply src/mcpil.py]: oh wait

Comment (leha-code) [reply src/mcpil.py]: jus a sec
dont merbe

**Comment by leha-code** [comment]
now u can merge after annoying code QL

**Comment by leha-code** [comment]
wait
again
ill add peoples splashes

**Comment by leha-code** [comment]
merge this

**Review by Alvarito050506 — APPROVED** [review]

**Comment by leha-code** [comment]
We will need a release on your birthdy lo' so evryone ill see.
Sorry fir this spelling thingi am tuping on a ebook

**Comment by leha-code** [comment]
Oops I scanned the code again and found that I accidentally left your b-day
as April's fools. Can you fix this?

**Pull request merged**